src/app/[locale]/not-found.tsx

```
@@ -0,0 +1,31 @@
+import Link from 'next/link'
+import { useTranslations } from 'next-intl'
+
+export default function NotFound() {
+  const t = useTranslations('error')
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)